internal-resources/git-workflow-and-contributor-guide.adoc

```diff
@@ -4,135 +4,226 @@
 :numbered:
 :internal-guide:
 
-= Stonesoup contributor guide
+= {Product_Name} Contributor Guide
 
-== Overview
+== How to use this guide
 
-This document describes the tasks needed to contribute to stonesoup documentation.
+The Contributor Guide is for writers on the {Product_Name} team, or anyone who needs to edit our documentation.
```

Comment:
Nitpick, but if we want to encourage engagement, perhaps we shouldn't confine it to anyone who **needs** to edit our documentation. 

what do you think about, `...or anyone who wants to edit or contribute to our documentation.`